boards/arm/deployment_module/deployment_module.dts

```diff
@@ -0,0 +1,267 @@
+            lookahead-size = <32>;
+            block-cycles = <(90 * 1000)>;
+        };
+    };
+
+	aliases {
+		led0 = &red_led_1;
+		led1 = &red_led_2;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+
+		red_led_0: led_0 {
+			gpios = <&gpiob 12 GPIO_ACTIVE_HIGH>;
+			label = "USER LED0";
+		};
+
+		red_led_1: led_1 {
+			gpios = <&gpiob 13 GPIO_ACTIVE_HIGH>;
+			label = "USER LED1";
+		};
+
+		red_led_2: led_2 {
+			gpios = <&gpiob 14 GPIO_ACTIVE_HIGH>;
+			label = "USER LED2";
+		};
+
+		red_led_3: led_3 {
+			gpios = <&gpiob 15 GPIO_ACTIVE_HIGH>;
+			label = "USER LED3";
+		};
+
+		wiz_led: wiz_led {
+			gpios = <&gpioc 7 GPIO_ACTIVE_HIGH>;
+			label = "Wiznet LED";
+		};
+	};
+
+	connector: gpios {
+		compatible = "gpio-keys";
+
+		gpio_0: gpios_0 {
+			gpios = <&gpioc 3 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
+			label = "CONNECTOR GPIO 0";
+		};
+
+		gpio_1: gpios_1 {
+			gpios = <&gpioc 4 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
+			label = "CONNECTOR GPIO 1";
+		};
+
+		gpio_2: gpios_2 {
+			gpios = <&gpioc 5 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
+			label = "CONNECTOR GPIO 2";
+		};
+
+		gpio_3: gpios_3 {
+			gpios = <&gpioa 6 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
+			label = "CONNECTOR GPIO 3";
+		};
+	};
+
+	pyro_sns: pyro_sns {
+		compatible = "gpio-keys";
+
+		pyro_sns_0: pyro_sns_0 {
+			gpios = <&gpioa 1 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
+			label = "PYRO SENSE CHANNEL 0";
+		};
+
+		pyro_sns_1: pyro_sns_1 {
+			gpios = <&gpioc 0 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
+			label = "PYRO SENSE CHANNEL 1";
+		};
+
+		pyro_sns_2: pyro_sns_2 {
+			gpios = <&gpioc 1 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
+			label = "PYRO SENSE CHANNEL 2";
+		};
+
+		pyro_sns_3: pyro_sns_3 {
+			gpios = <&gpioc 2 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
+			label = "PYRO SENSE CHANNEL 3";
+		};
+	};
+
+	pyro_ctrl: pyro_ctrl {
+		compatible = "gpio-keys";
+
+		pyro_ctrl_0: pyros_ctrl_0 {
+			gpios = <&gpioa 11 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
+			label = "PYRO CTRL CHANNEL 0";
+		};
+
+		pyro_ctrl_1: pyros_ctrl_1 {
+			gpios = <&gpioa 12 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
+			label = "PYRO CTRL CHANNEL 1";
+		};
+
+		pyro_ctrl_2: pyros_ctrl_2 {
+			gpios = <&gpioc 9 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
+			label = "PYRO CTRL CHANNEL 2";
+		};
+
+		pyro_ctrl_3: pyros_ctrl_3 {
+			gpios = <&gpioc 8 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
+			label = "PYRO CTRL CHANNEL 3";
+		};
+	};
+};
+
+&clk_lsi {
+	status = "okay";
+};
+
+&clk_hsi {
+	clock-frequency = <DT_FREQ_M(16)>;
+	status = "okay";
+};
+
+&rcc {
+	clocks = <&clk_hsi>;
+	clock-frequency = <DT_FREQ_M(16)>;
+	ahb-prescaler = <1>;
+	apb1-prescaler = <2>;
+	apb2-prescaler = <1>;
+};
+
+&usart2 {
+	pinctrl-0 = <&usart2_tx_pa2 &usart2_rx_pa3>;
+
+	pinctrl-names = "default";
+	current-speed = <115200>;
+	status = "okay";
+};
+
+&uart5 {
+	pinctrl-0 = <&uart5_tx_pc12 &uart5_rx_pd2>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+	status = "okay";
+};
+
+&spi1 {
```

Comment:
should there be a w25q128jvsiq in here for the FS? off your question yesterday im realizing we dont really  need it on this mod but its in the schematoc

Reply:
You might be looking at the wrong one. This is for Mary's deployment module which doesn't have one. We will be using the onboard flash the micro provides.

Reply:
oh ya i super was